config/routes.rb

```diff
@@ -8,7 +8,9 @@
   
   namespace :api do
     namespace :v1 do
-      resources :artists, only: [:index]
+      resources :artists, only: [:index, :show]
     end
   end
+
+  get "*path", to:"homes#index", via: :all
```

Comment:
as mentioned earlier, I dont want folks using the wild-card path. Instead, for any new paths you should add, you should be adding those explicitly